doc/source/serve/doc_code/monitoring/request_id.py

```diff
@@ -0,0 +1,17 @@
+from ray import serve
+
+import requests
+
+
+@serve.deployment
+class Model:
+    def __call__(self) -> int:
+        return 1
+
+
+serve.run(Model.bind())
+resp = requests.get(
+    "http://localhost:8000", headers={"RAY_SERVE_REQUEST_ID": "123-234"}
```

Comment:
Are there any limitations on the request id's format (e.g. alphanumeric)? Is there any undefined behavior if two requests share the same id?

Reply:
No, we don't have limitations. Currently the request can be any chars string. When user setting the request id, relying on the customer to make it as unique, there is no undefined behavior, but you will see duplicate request id in the log.

Reply:
Sounds good!